src/main/java/com/minecolonies/coremod/blocks/AbstractBlockHut.java

```diff
@@ -237,7 +238,8 @@ public IBlockState getStateForPlacement(
      * @param state   the state the placed block is in.
      * @param placer  the player placing the block.
      * @param stack   the itemstack from where the block was placed.
-     * @see Block#onBlockPlacedBy(World, BlockPos, IBlockState, EntityLivingBase, ItemStack)
+     * @see Block#onBlockPlacedBy(World, BlockPos, IBlockState,
```

Comment:
these breakups are unneeded, please break after 120 or 180 not before. And please in other cases break manually not by formatter